part-1-ajax/test.js

```diff
@@ -1,76 +1,91 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
 
   var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
 
+  button.addEventListener('click', function(event){
+    event.preventDefault()
+    var s = $('input').val()
+    $('input').val('')
     $.ajax({
```

Comment:
nice job! in the future you want to separate out function whenever possible. So instead of having one function that is 25 lines. Have multiple ones that take up fewer lines.